doc/gettingstarted.md

```diff
@@ -109,10 +109,17 @@ You can now register an OTAA activated device to be used with the stack as follo
 $ ttn-lw-cli end-devices create app1 dev1 --dev-eui 0004A30B001C0530 --join-eui 800000000000000C --frequency-plan-id EU_863_870 --root-keys.app-key.key 752BAEC23EAE7964AF27C325F4C23C9A --lorawan-phy-version 1.0.2-b --lorawan-version 1.0.2
 ```
 
-This will create an LoRaWAN 1.0.2 end device `dev1` with DevEUI `0004A30B001C0530`, AppEUI `800000000000000C` and AppKey
- `752BAEC23EAE7964AF27C325F4C23C9A`. After configuring the credentials in the end device, you should be able to join the private network.
+This will create an LoRaWAN 1.0.2 end device `dev1` with DevEUI `0004A30B001C0530`, AppEUI `800000000000000C` and AppKey `752BAEC23EAE7964AF27C325F4C23C9A`. After configuring the credentials in the end device, you should be able to join the private network.
 
-If you wish to enable class C support for this device, you can add the `--supports-class-c` flag in the above command.
+It is also possible to register an ABP activated device using the `--abp` flag as follows:
+
+```bash
+$ ttn-lw-cli end-devices create app1 dev1 --frequency-plan-id EU_863_870 --lorawan-phy-version 1.0.2-b --lorawan-version 1.0.2 --abp --session.dev-addr 00E4304D --session.keys.app-s-key.key A0CAD5A30036DBE03096EB67CA975BAA --session.keys.f_nwk_s_int_key.key b7f3e161bc9d4388e6c788a0c547f255
+```
+
+This will create an LoRaWAN 1.0.2 end device `dev1` with DevAddr `00E4304D`, AppSKey `A0CAD5A30036DBE03096EB67CA975BAA` and NwkSKey `B7F3E161BC9D4388E6C788A0C547F255`.
+
+If you wish to enable class C support for these devices, you can add the `--supports-class-c` flag in the above commands.
```

Comment:
I think we should clarify this a bit more, or move it to the multicast (or separate class C) section.

Thing is that if people set this flag, the downlink scheduling is class C.